examples/darwin-framework-tool/entitlements/codesign.py

```diff
@@ -34,7 +34,8 @@ def get_identity():
             "No valid identity has been found. Application will run without entitlements.")
         exit(0)
 
-    identity = re.search(r'\b[0-9a-fA-F]{40}\b', command_result)
+    command_result = command_result.replace("\\n", "\n")
+    identity = re.search(r'\b[0-9a-fA-F]{40}\b(?![^\n]*\(CSSMERR_TP_CERT_EXPIRED\))', command_result)
```

Comment:
Unrelated but I think the code sign script from this app can be removed at the moment.

Reply:
@krypton36 See above?

Reply:
Yes, it can be removed. Darwin-framework-tool no longer requires code signing since it does not use keychain.